File: iosApp/iosApp/Presentation/Auth/ResetPassword/ResetPasswordUiState.swift
//
//  ResetPasswordUiState.swift
//  ConCafe
//
//  Created by 홍희표 on 2026/03/28.
//

import Foundation

struct ResetPasswordUiState {
    var email: String

    var isSubmitting: Bool

    static let empty = ResetPasswordUiState(
        email: "",
        isSubmitting: false
    )
}

File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/auth/resetpassword/ResetPasswordUiState.kt
package com.hhp227.concafe.presentation.auth.resetpassword

data class ResetPasswordUiState(
    val email: String,
    val isSubmitting: Boolean
) {
    companion object {
        fun empty(): ResetPasswordUiState {
            return ResetPasswordUiState(
                email = "",
                isSubmitting = false
            )
        }
    }
}
